e2e/e2e-example.sh

```diff
@@ -104,6 +109,8 @@ check_status "${envoy_url_echo}" 200 -X POST -H 'Content-Type: application/x-www
 ((step+=1))
 echo "[${step}/${total_steps}] (onRequestBody) Testing true positive request (body)"
 check_status "${envoy_url_unfiltered}" 403 -X POST -H 'Content-Type: application/x-www-form-urlencoded' --data "${truePositiveBodyPayload}"
+# Every true positive request should be denied with a 403 and an empty body
+check_body "${envoy_url_filtered}" true
```

Comment:
I don't think this works as expected. We are not testing the same request we check the status on.

Reply:
My bad! The URL should be changed accordingly, but I removed these extra checks. As agreed offline, a more comprehensive go code for e2e tests will follow.